apps/vite/src/pages/Home.tsx

```
@@ -1,57 +1,58 @@
 
 function Home() {
-  const { BSYear, BSMonth, pageType = "calendar" } = useParams()
+  const { BSYear, BSMonth, pageType = 'calendar' } = useParams();
 
   const validYearAndMonth = useMemo(() => {
-    if (!BSYear || !BSMonth) return new NepaliDate()
-    const year = parseInt(BSYear)
-    const month = parseInt(BSMonth)
-    const isValid = year >= 2075 && year <= 2082 && month >= 1 && month <= 12
+    if (!BSYear || !BSMonth) return new NepaliDate();
+    const year = parseInt(BSYear);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)